backend/src/main/java/net/es/oscars/topo/pop/TopoPopulator.java

```diff
@@ -158,6 +163,14 @@ public Topology loadTopology() throws TopoException, ResourceAccessException, IO
                     .build();
             devices.put(d.getUrn(), d);
             for (OscarsOnePort discPort : discDevice.getPorts()) {
+                // If this is an 'untagged' port, and features.untagged_ports is disabled,
+                // just skip this port.
+                if (
+                    featuresProperties.getUntaggedPorts() == false
+                    && discPort.getEthernetEncapsulation() == EthernetEncapsulation.NULL
```

Comment:
this condition would allow ports with `EthernetEncapsulation.QINQ` to sneak in (not that we have any, but if we did, we couldn't support them) 

instead, do `discPort.getEthernetEncapsulation() != EthernetEncapsulation.DOT1Q`